CRM/Core/EntityTokens.php

```diff
@@ -229,7 +229,7 @@ public function isCustomField(string $fieldName) : bool {
    * @return bool
    */
   public function isMoneyField(string $fieldName): bool {
-    return $this->getFieldMetadata()[$fieldName]['data_type'] === 'Money';
+    return ($this->getFieldMetadata()[$fieldName]['data_type'] ?? '') === 'Money';
```

Comment:
enotice - I wanted to fix the metadata not the notice - but I feel like I need to get this change through before I can focus in on the metadata